README.md

```diff
@@ -130,18 +130,14 @@ Here are the prerequisites to build the keyple components (jars)
 #### Linux or Macos
 Following commands will build all the artifacts at once. The first command is required to be executed at least once to build the gradle wrapper. Then, the keyple-core artifact is built and installed into the local maven repository as other artifacts depends on it.  
```

Comment:
The part: _The first command is required to be executed at least once to build the gradle wrapper_ can be deleted, has the wrapper is now included